bin/node/runtime/src/lib.rs

```diff
@@ -850,6 +850,8 @@ impl_runtime_apis! {
 			repeat: u32,
 		) -> Result<Vec<frame_benchmarking::BenchmarkResults>, sp_runtime::RuntimeString> {
 			use frame_benchmarking::Benchmarking;
+			use pallet_session_benchmarking::Module as SessionBench;
```

Comment:
This seems out of place, why can't we use the normal `Session` which is already in scope? I suppose it is linked to the dependency of staking and Session. A note explaining why it is the way it is would be good.

Reply:
Yeah, I had to do some special tricks here because of cyclic dependencies. A comment is a good idea.